index.js

```diff
@@ -121,13 +123,28 @@ function onchange (match, command, rawArgs, opts = {}) {
   // Create the "watcher" instance for file system changes.
   const watcher = chokidar.watch(matches, {
     cwd: cwd,
-    ignored: opts.exclude || [],
+    ignored: getIgnoreMergedFromIgnoreFile(opts.ignorePath),
```

Comment:
This breaks `opts.exclude`

Reply:
Not really, if you see the function you will realize that there is a merge between the excludes and the ignorefile.

Reply:
The code is a bit unclear, I didn't see it was nested inside this function. Can you move the functionality outside of this function so it can be tested independently?

Reply:
Yes, I can.

Reply:
One moment

Reply:
Ready. That's better?